raiden/blockchain/state.py

```diff
@@ -53,8 +51,12 @@
 class ChannelSettleState:
     """Recovered channel state that corresponds to the on-chain data."""
 
+    # XXX backwards compatibility database?
```

Comment:
@ulope?

Reply:
Adding fields without default values to state that can be in the database makes old database content incompatible with the node code.

Unless I'm mistaken that doesn't seems to be relevant here though since the `ChannelSettleState` isn't a true `State` that can end up in the DB.